cmd/osbuild-worker/main.go

```diff
@@ -261,6 +263,14 @@ func RunJob(job worker.Job, store string, kojiServers map[string]koji.GSSAPICred
 				}
 			}()
 
+			if result.Success == false {
```

Comment:
Why are all the others `!result.Success` and this `result.Success == false`? ;P

Reply:
The linter complained, too :)